app/javascript/components/sponsor_us/SponsorCard.jsx

```diff
@@ -10,10 +10,13 @@ const SponsorCard = ({ type, amount, returns, icon }) => {
     return (
         <Card className="sponsor-card w-[22rem]">
             <div className="flex flex-row items-start">
-                <Icon className="h-14" />
+                <Icon className={type === 'Ruby' ? 'h-[58px] mt-[-5px]' : 'h-14'} />
                 <div className="flex flex-col flex-1 ml-5">
                     <h3 className="text-xl font-bold">{type} Sponsor</h3>
-                    <p className={`text-lg ${type.toLowerCase()}-color`}>${amount}</p>
+                    <div className="flex flex-row items-center">
+                        <p className={`text-lg ${type.toLowerCase()}-color`}>${amount}</p>
+                        <p className="text-sm ml-2 text-wnbrb-gray-medium">/6 months</p>
```

Comment:
I added some text here specifying that the sponsorship is for 6 months because that wasn't clear before.

<img width="349" alt="Screen Shot 2022-02-21 at 11 14 04 AM" src="https://user-images.githubusercontent.com/9601737/154992081-a01678b2-98b7-4311-ad83-81aee8cd4962.png">